lapce-data/src/config.rs

```diff
@@ -128,12 +130,21 @@ pub struct EditorConfig {
     pub tab_width: usize,
     #[field_names(desc = "If opened editors are shown in a tab")]
     pub show_tab: bool,
+    #[field_names(
+        desc = "How long (in ms) it should take before the hover information appears"
+    )]
+    #[serde(default = "EditorConfig::default_hover_delay")]
```

Comment:
I guess this is to save the need to put the field in the default settings file. But I would prefer to remove this and put it in the settings file because that default file is a kind of documentation that lists all the settings.